CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+## How to contribute code
+
+Follow these steps to submit your code contribution.
+
+### Step 1. Open an issue
+
+Before making any changes, we recommend opening an issue (if one doesn't already
+exist) and discussing your proposed changes. This way, we can give you feedback
+and validate the proposed changes.
+
+If the changes are minor (simple bug fix or documentation fix), then feel free
+to open a PR without discussion.
+
+### Step 2. Make code changes
+
+To make code changes, you need to fork the repository. You will need to setup a
+development environment and run the unit tests. This is covered in section
+"Setup environment".
+
+### Step 3. Create a pull request
+
+Once the change is ready, open a pull request from your branch in your fork to
+the master branch in [keras-team/keras](https://github.com/keras-team/keras).
+
+### Step 4. Sign the Contributor License Agreement
+
+After creating the pull request, the `google-cla` bot will comment on your pull
+request with instructions on signing the Contributor License Agreement (CLA) if
+you haven't done so. Please follow the instructions to sign the CLA. A `cla:yes`
+tag is then added to the pull request.
+
+![Tag added](https://i.imgur.com/LHEdIfL.png)
+
+
+### Step 5. Code review
+
+CI tests will automatically be run directly on your pull request.  Their
+status will be reported back via GitHub actions.
+
+There may be
+several rounds of comments and code changes before the pull request gets
+approved by the reviewer.
+
+![Approval from reviewer](https://i.imgur.com/zgRziTt.png)
+
+### Step 6. Merging
+
+Once the pull request is approved, a team member will take care of merging.
+
+## Setup environment
+
+Setting up your KerasCV development environment is quite easy.  You simply
+need to run the following commands:
+
+```shell
+git clone https://github.com/YOUR_GITHUB_USERNAME/keras-cv.git
```

Comment:
I would maybe just move "forking and cloning" into the description above rather than these lines. There's not a universal way to clone (ssh vs https, etc).

Reply:
Yeah, there isn't a universal way.  That being said, I think having a set of copy pasteable commands still serves value.  

A user who is concerned about the nuance between ssh/https cloning will probably be able to read the command and use their desired approach.  Thoughts?

Reply:
If you want copy pastable, one option is to go with the `gh` command line tool? `gh repo fork keras-team/keras-cv --clone --remote` is a one liner that would handle forking, cloning and adding a remote in one go, and will trigger an auth flow if not already set up I think? It's also copy pastable without edits.

https://cli.github.com/manual/gh_repo_fork

Reply:
Wow - that's a really nice command.  Thanks

Reply:
I hope that we could point on a command that is already installed also on our Docker devel image (and our `.devcontainer`).

Then for extra things we could have a link to:
 https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository

Reply:
I think that we need at least to enforce the minimal for the linting/formatting tools at:
https://github.com/keras-team/keras-cv/blob/master/setup.py#L29